skops/io/tests/test_persist.py

```diff
@@ -145,9 +157,10 @@ def _tested_estimators(type_filter=None):
                     # scikit-learn < 1.4.0) is not available in scipy >= 1.11.0. The
                     # default solver will be "highs" from scikit-learn >= 1.4.0.
                     # https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.QuantileRegressor.html
-                    estimator = _construct_instance(partial(Estimator, solver="highs"))
+                    estimator = construct_instances(partial(Estimator, solver="highs"))
```

Comment:
I think we should accept all instances returned by this (new) function and like sklearn, test all of them, which gives us more coverage.

Reply:
there seems to be no issues with coverage once the tests pass